tests/framework/PostProcessors/BasicStatistics/gold/basicStatisticsTimeDependentAsynchronousHistories/HistorySetPostProcTestSyncronized_dump_3.csv

```diff
@@ -1,12 +1,12 @@
-y,x,z,time
```

Comment:
It seems to me this file does not need to be regolded.

Reply:
you are right.